### DISCUSSION TITLE
Bind arrowkeys to targets in the browser calc

### DISCUSSION THREAD

**Discussion by Avslutarn** [Endebert/squadmc#108] [Ideas]
Hello!

I use the Post Scriptum browser calculator all the time and when I want to use the line or area function I want to select the next point quickly and efficiently. This is however hard to do while under time pressure and wanting to go fast. This results in either clicking outside of next target or clicking the line which prompts the new target dialoge pop up instead of selecting the next target pin.

There are arrows in the bottom of the screen where you can select "next" target which is good, but an addition to this could be to bind the left and right arrow keys so that you can tab out to the browser, press right arrow to go to the next target and then tab back in. It is highly accurate, fast and if you go too far forward you can always use the opposite arrow key to go "back". No need to try and drag your cursor over to the other screen and pin point target a small button or target.

**Comment by gdvdialoge** [comment]
Das Portal ermöglicht einen strukturierten Zugriff auf wichtige Versicherungsdaten und digitale Services. Nutzer profitieren von einer klaren Übersicht und einfachen Navigation durch die verschiedenen Funktionen. Moderne Sicherheitsstandards schützen alle sensiblen Informationen zuverlässig. In diesem Zusammenhang spielt das <a href="https://gdv-dialoge2002.de/"> GDV Dialoge Datenportal</a>eine zentrale Rolle bei der digitalen Verwaltung.
